lib/gis/parser_standard_options.c

```diff
@@ -652,6 +653,22 @@ struct Option *G_define_standard_option(int opt)
         Opt->description =
             _("Either a standard color name, R:G:B triplet, or \"none\"");
         break;
+    case G_OPT_C_FORMAT:
+        Opt->key = "color_format";
+        Opt->type = TYPE_STRING;
+        Opt->key_desc = "name";
+        Opt->required = YES;
+        Opt->multiple = NO;
+        Opt->answer = "hex";
+        Opt->options = "rgb,hex,hsv,triplet";
+        Opt->label = _("Color format");
+        Opt->description = _("Color format for output values.");
+        G_asprintf(
+            (char **)&(Opt->descriptions), "rgb;%s;hex;%s;hsv;%s;triplet;%s",
+            _("output color in RGB format"), _("output color in HEX format"),
+            _("output color in HSV format"),
```

Comment:
Can you please add the note about experimental to the CLI API or doc? Something like this in CLI is perhaps the way to go?

```suggestion
            _("output color in HSV format (experimental)"),
```

Reply:
Sure, should I add this here only, or should it be included somewhere else as well?